packages/dd-trace/src/plugins/util/web.js

```diff
@@ -476,7 +476,8 @@ function addResponseTags (context) {
   const { req, res, paths, span, inferredProxySpan } = context
 
   if (paths.length > 0) {
-    span.setTag(HTTP_ROUTE, paths.join(''))
+    const pathStr = paths.join('')
+    if (pathStr.length > 0) span.setTag(HTTP_ROUTE, paths.join(''))
   }
```

Comment:
```suggestion
  const route = paths.join('')
  if (route) {
    span.setTag(HTTP_ROUTE, route)
  }
```

Nit: that way join is only called once and I expect the average case to be that paths contains at least one entry. So we would always do the length check anyway.